docs/data-sources/relation_from_taboo_contract_to_filter.md

```diff
@@ -0,0 +1,58 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# In order to regenerate this file execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+subcategory: "Contract"
+layout: "aci"
+page_title: "ACI: aci_relation_from_taboo_contract_to_filter"
+sidebar_current: "docs-aci-data-source-aci_relation_from_taboo_contract_to_filter"
+description: |-
+  Data source for ACI Relation From Taboo Contract To Filter
+---
+
+# aci_relation_from_taboo_contract_to_filter #
+
+Data source for ACI Relation From Taboo Contract To Filter
```

Comment:
```suggestion
Data source for ACI Relation From Taboo Contract Subject To Filter
```
Are we missing the Deny part? What does that filter relationship actually do?

Reply:
It specifies a relationship to a filter that indicates what traffic is denied to the EPG. The naming is set because the MO is Taboo Contract Subject and the relationship points to the filter. I could add deny in front of filter if that makes it more clear, but not sure we should specify this in our naming when UI is also not doing this.